database/queries.sql

```diff
@@ -0,0 +1,37 @@
+--Fetch all the favourite items for a given user
+SELECT i.name
+FROM onboarding_schema.users as u, onboarding_schema.mark_as_favourite as fav, onboarding_schema.items i
```

Comment:
you missed the "as" in the last one, its not necessary but lets keep them all with or without "as", however you prefer

Also I think I would go with a JOIN instead of adding the conditions in the WHERE, and change u.name to u.id since names are not unique
```
SELECT i.name
FROM onboarding_schema.users as u JOIN
 onboarding_schema.mark_as_favourite as fav ON u.id = fav.user JOIN
onboarding_schema.items as i ON fav.item = i.id 
WHERE u.id = 1;
```